Basement content

vinnn-commit opened the pull request.

vinnn-commit committed:
Update Casual Corridor Walk.py

vinnn-commit committed:
Update Casual Corridor Walk.py

vinnn-commit committed:
help

vinnn-commit committed:
Update convo with cleaner

vinnn-commit committed:
Update on basement

amm0nition reviewed (changes requested):
Please fix grammatical and punctuational errors in the dialogues.

vinnn-commit committed:
2nd floor and basement update

basement is completely done, second floor whis is the balcony is still on progress
